Docker build for Wildfly Distro broken since beta-4

**Describe the bug**

The Docker Image for the beta-4 release for the Wildfly distribution was not successfully published. Subsequent builds of beta-5-SNAPSHOT also fail

**To Reproduce**

See the build: https://github.com/operaton/operaton-docker/actions/runs/14898191548/job/41844807962

**Additional context**
I created the issue here, as it is not yet clear if it is an overall build problem with the distribution or with the operaton-docker project, and the visibility here is higher

Can replicate locally, Wildfly Integration Tests are broken for h2 and postgres. A module seems to be missing:

```
Caused by: org.jboss.modules.xml.XmlPullParserException: Failed to add resource root 'java-uuid-generator-4.3.0.jar' at path 'java-uuid-generator-4.3.0.jar' (position: END_TAG seen ...ources>\n    <resource-root path="java-uuid-generator-4.3.0.jar" />... @3:59) caused by: java.nio.file.NoSuchFileException: /Users/zoeller/sources/operaton/operaton-bpm-platform/qa/wildfly-runtime/target/server/wildfly-35.0.1.Final/modules/com/fasterxml/uuid/java-uuid-generator/main/java-uuid-generator-4.3.0.jar
	at org.jboss.modules.xml.ModuleXmlParser.parseResourceRoot(ModuleXmlParser.java:1120)
	at org.jboss.modules.xml.ModuleXmlParser.parseResources(ModuleXmlParser.java:931)
	at org.jboss.modules.xml.ModuleXmlParser.parseModuleContents(ModuleXmlParser.java:696)
	at org.jboss.modules.xml.ModuleXmlParser.parseDocument(ModuleXmlParser.java:468)
	at org.jboss.modules.xml.ModuleXmlParser.parseModuleXml(ModuleXmlParser.java:334)
	... 46 more
Caused by: java.nio.file.NoSuchFileException: /Users/zoeller/sources/operaton/operaton-bpm-platform/qa/wildfly-runtime/target/server/wildfly-35.0.1.Final/modules/com/fasterxml/uuid/java-uuid-generator/main/java-uuid-generator-4.3.0.jar
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixFileAttributeViews$Basic.readAttributes(UnixFileAttributeViews.java:55)
	at java.base/sun.nio.fs.UnixFileSystemProvider.readAttributes(UnixFileSystemProvider.java:149)
	at java.base/java.nio.file.Files.readAttributes(Files.java:1851)
	at java.base/java.util.zip.ZipFile$Source.get(ZipFile.java:1264)
	at java.base/java.util.zip.ZipFile$CleanableResource.<init>(ZipFile.java:709)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:243)
	at java.base/java.util.zip.ZipFile.<init>(ZipFile.java:172)
	at java.base/java.util.jar.JarFile.<init>(JarFile.java:347)
	at org.jboss.modules.xml.ModuleXmlParser$DefaultResourceRootFactory.createResourceLoader(ModuleXmlParser.java:1622)
	at org.jboss.modules.LocalModuleFinder.lambda$new$0(LocalModuleFinder.java:103)
	at org.jboss.modules.xml.ModuleXmlParser.parseResourceRoot(ModuleXmlParser.java:1118)
	... 50 more
```

--- a/distro/wildfly/modules/src/main/modules/com/fasterxml/uuid/java-uuid-generator/main/module.xml
+++ b/distro/wildfly/modules/src/main/modules/com/fasterxml/uuid/java-uuid-generator/main/module.xml
@@ -1,6 +1,6 @@
 <module xmlns="urn:jboss:module:1.0" name="com.fasterxml.uuid.java-uuid-generator">
   <resources>
-    <resource-root path="java-uuid-generator-4.3.0.jar" />
+    <resource-root path="java-uuid-generator-5.1.0.jar" />
   </resources>
 
   <dependencies>

